README.org

```diff
@@ -13,150 +13,75 @@ As Python 3 matures and majority python packages support Python 3, the scientifi
   micronota can annotate multiple features including coding genes, prophage, CRISPR, tRNA, rRNA and other ncRNAs. It has a customizable framework to integrate additional tools and databases. Generally, the annotation can be classified into 2 categories: structural annotation and functional annotation. Structural annotation is the identification of the genetic elements on the sequence and functional annotation is to assign functions to those elements.
 
 * Install
-
-  To install the latest release of micronota:
+** install via conda
+   To install the latest release of micronota:
   #+BEGIN_SRC sh
     conda install micronota
   #+END_SRC
-
-
-  Or you can install through ~pip~:
+** install via pip
+   You can install through ~pip~:
   #+BEGIN_SRC sh
     pip install micronota
   #+END_SRC
 
-  To install the latest developping version:
+  To install or update to the latest developping version:
   #+BEGIN_SRC
     pip install git+git://github.com/biocore/micronota.git
   #+END_SRC
 
-* Prepare Databases
-
-  To prepare (download and format) the files of TIGRFAM to the right form read by micronota:
-  #+BEGIN_SRC
-    micronota database prepare tigrfam --cache_dir ~/database
-  #+END_SRC
-* Print Configure Info
-
-  To check the micronota setup, you can run:
+  micronota relies on external packages like prodigal, aragorn, etc. for annotation. You need to install those external packages (this is implicitly installed when micronota is installed via conda). To simplify the process, you can install with conda:
   #+BEGIN_SRC
-    micronota info
-  #+END_SRC
-
-  It will print out the system info, databases available, external tools, and other configuration info.
-
-* Config Files
-
-  There are 3 types of config files user can set for workflow, logging, and parameters, respectively. All of the 3 config files can be specified on the command line (=--cfg=, =--log=, and =--param=) to override the default settings. Besides, you can also put the frequently used settings into global config files. The global config files should be put in the config directory of micronota. On linux it is usually =~/.config/micronota=; on Mac, it is usually =~/Library/Application Support/micronota=. You can also find the directory by running the following command
-  #+BEGIN_SRC python
-    python -c "import click; print(click.get_app_dir('micronota'))"
+    conda install --file https://raw.githubusercontent.com/biocore/micronota/master/ci/conda_requirements.txt
```

Comment:
Do you expect certain versions of each tool? Are the versions logged somewhere for future reference?